Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx

```diff
@@ -206,6 +214,32 @@ class StatusTaskModule extends React.Component<RouteComponentProps, IStatusState
             return (<div></div>);
         }
     }
+
+    private renderErrorMessage = () => {
+        if (this.state.message.errorMessage) {
+            return (
+                <div>
+                    <h3>Error message</h3>
+                    <span>{this.state.message.errorMessage}</span>
```

Comment:
The warning and error messages could be lengthy. 
I think it's safe to use an element with scroll bar, instead of a span.

Reply:
There is a scrollbar on parent div container so that the long text won't reflect layout.